src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -238,6 +253,44 @@ function reducer(draft, action) {
             draft.formSteps = [...updatedSteps, ...draft.formSteps.slice(index+1)];
             break;
         }
+        /**
+         * Form Logic rules actions
+         */
+        case 'LOADED_RULES': {
+            draft.rules.rules = action.payload.length ? action.payload : [{
+                ...EMPTY_RULE,
+                form: draft.form.url
+            }];
```

Comment:
The empty-rule-by-default was mostly a debug thing so I had one less button to click. I don't think we should start with an empty rule, as saving a new form without focusing the logic tab would then lead to invalid rules being created and thus frustration for the end-user. Let's just keep an empty list as default.